### ISSUE TITLE
Improved twisted website content

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10449]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)| @puzzlement reported|
|-|-|
|Trac ID|trac#697|
|Type|enhancement|
|Created|2004-08-30 06:18:24Z|

<details><summary>Searchable metadata</summary>

```
trac-id__697 697
type__enhancement enhancement
reporter__hypatia hypatia
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__website website
keywords__documentation documentation
time__1093846704000000 1093846704000000
changetime__1128845970000000 1128845970000000
version__ 
owner__ 
cc__hypatia
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
There's some consensus that the website needs to be more "useful".

Ideas (care of Itamar):
 - website should be designed in terms of "what do you want to do?" -- ie do you
want a web server, ftp client, ssh... ?
 - projects should be clearly delineated
 - each section should have pointers to overview, capabilities, and general
documentation

More notes to follow... (I think)
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Project websites themselves (especially for web) should exist.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Ok, I think this one was fixed with the new website for 2.0.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
